wd_helper.au3

```diff
@@ -1610,6 +1610,91 @@ Func _WD_GetBrowserPath($sBrowser)
 	Return SetError(__WD_Error($sFuncName, $iErr, Default, $iExt), $iExt, $sPath)
 EndFunc   ;==>_WD_GetBrowserPath
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_GetBrowserPID
+; Description ...: Get the PID of the browser that was launched by WebDriver
+; Syntax ........: _WD_GetBrowserPID($iWebDriverPID, $sBrowserName)
+; Parameters ....: $iWebDriverPID       - WebDriver PID returned by _WD_Startup()
+;                  $sBrowserName        - [optional] Browser name from the list of supported browsers ($_WD_SupportedBrowsers)
+; Return values .: Success - Browser PID
+;                  Failure - 0 and sets @error to one of the following values:
+;                  - $_WD_ERROR_GeneralError
+;                  - $_WD_ERROR_NotSupported
+;                  - $_WD_ERROR_NoMatch
+; Author ........: mLipok
+; Modified ......: Danp2
+; Remarks .......:
+; Related .......:
+; Link ..........:
+; Example .......: No
+; ===============================================================================================================================
+Func _WD_GetBrowserPID($iWebDriverPID, $sBrowserName = '')
+	Local Const $sFuncName = "_WD_GetBrowserPID"
+	Local $iErr = $_WD_ERROR_Success, $iExt = 0, $iIndex = 0, $sBrowserExe = '', $aProcessList, $iBrowserPID = 0, $sMessage = ''
+	Local $sDriverProcessName = _WinAPI_GetProcessName($iWebDriverPID)
```

Comment:
There's an issue with _WinAPI_GetProcessName ([trac issue](https://www.autoitscript.com/trac/autoit/ticket/3894)). In the interim, we should add a check for ProcessExists().